src/frontend/context/FagsakContext.tsx

```diff
@@ -26,9 +26,9 @@ const [FagsakProvider, useFagsak] = createUseContext(() => {
             .then((hentetFagsak: Ressurs<IFagsak>) => {
                 settFagsak(hentetFagsak);
             })
-            // eslint-disable-next-line @typescript-eslint/no-unused-vars
+
             .catch((_error: AxiosError) => {
-                settFagsak(byggFeiletRessurs('Ukjent feil ved henting av fagsak'));
+                settFagsak(byggFeiletRessurs('Ukjent feil ved henting av fagsak', _error.status));
```

Comment:
"_"-prefixen var vel satt fordi variabelen ikke var i bruk? Greit å fjerne den når vi først bruker den :D 
```suggestion
                settFagsak(byggFeiletRessurs('Ukjent feil ved henting av fagsak', error.status));
```